charts/mysql-operator/templates/statefulset.yaml

```diff
@@ -49,7 +49,7 @@ spec:
           args:
             - --leader-election-namespace={{ .Release.Namespace }}
             # connect to orchestrator on localhost
-            - --orchestrator-uri=http://127.0.0.1:3000/api
+            - --orchestrator-uri=http://{{ template "mysql-operator.fullname" . }}.{{ .Release.Namespace }}/api
```

Comment:
Why pointing to the orchestrator service and not leting to connect to the local orchestrator instance?

Reply:
https://github.com/presslabs/mysql-operator/blob/master/pkg/controller/mysqlcluster/internal/syncer/statefullset.go#L324-L327

Because the sidecar in MySQL will use this API (e.g. in preStop), if you use localhost directly, these components will not be accessible